### PR TITLE
Modify ocpv-demo for Ansible validated content

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://github.com/ansible-middleware/ocpv-demo/issues/1

[Don't merge now, few works left]

### CODE REVIEW

**Comment by RanabirChakraborty** [comment]
@sabre1041 can you please review this?
I'm unsure about the role description so I left it blank, Let me know what to fill up. 
We need to change the repo name to `ocpv-lab`; I have renamed it accordingly.
We also need to release `ocpv-lab` to create a `changelog`.

File: README.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
-# Ansible Middleware OpenShift Virtualization Demo
+# Ansible Middleware OpenShift Virtualization lab
```
Comment (sabre1041) [inline README.md]:
```suggestion
# Ansible Middleware OpenShift Virtualization Lab
```

File: README.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
 
-Demonstrating the use of OpenShift Virtualization as an environment for hosting Virtual Machines that are provisioned, configured and maintained using Ansible and tooling from the Ansible Middleware project.
+Using of OpenShift Virtualization as an environment for hosting Virtual Machines that are provisioned, configured and maintained using Ansible and tooling from the Ansible Middleware project.
```
Comment (sabre1041) [inline README.md]:
```suggestion
Using OpenShift Virtualization as an environment for hosting Virtual Machines that are provisioned, configured and maintained using Ansible and tooling from the Ansible Middleware project.
```

File: roles/configure-aap/README.md [hunk 3]
```diff
@@ -11,28 +11,52 @@ Any pre-requisites that may not be covered by Ansible itself or the role should
...
-
-    - hosts: servers
-      roles:
-         - { role: username.rolename, x: 42 }
-
 License
 -------
 
-BSD
+[GPL2](https://github.com/ansible-middleware/ocpv-lab/blob/validated_content/LICENSE)
```
Comment (sabre1041) [inline roles/configure-aap/README.md]: Can we change the license to Apache 2?

File: roles/ocpv-install/README.md [hunk 4]
```diff
@@ -11,28 +11,33 @@ Any pre-requisites that may not be covered by Ansible itself or the role should
...
-
-    - hosts: servers
-      roles:
-         - { role: username.rolename, x: 42 }
-
 License
 -------
 
-BSD
+[GPL2](https://github.com/ansible-middleware/ocpv-lab/blob/validated_content/LICENSE)
```
Comment (sabre1041) [inline roles/ocpv-install/README.md]: Change to Apache 2

**Review by sabre1041 — CHANGES_REQUESTED** [review]

**Comment by RanabirChakraborty** [comment]
@sabre1041 the changes are done. Can you please review once.

File: roles/configure-aap/README.md [hunk 5]
```diff
@@ -1,38 +1,62 @@
...
-      roles:
-         - { role: username.rolename, x: 42 }
+Please follow [configure-aap.yml](https://github.com/ansible-middleware/ocpv-lab/blob/main/playbooks/configure-aap.yml)
+for better understanding.
 
 License
 -------
 
-BSD
+[Apache License Version 2.0](https://github.com/ansible-middleware/ocpv-lab/blob/validated_content/LICENSE)
```
Comment (sabre1041) [inline roles/configure-aap/README.md]:
```suggestion
[Apache License Version 2.0](https://github.com/ansible-middleware/ocpv-lab/blob/main/LICENSE)
```

**Review by sabre1041 — CHANGES_REQUESTED** [review]

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**